api/envoy/type/v3/http_status.proto

```diff
@@ -95,6 +95,8 @@ enum StatusCode {
 
   ExpectationFailed = 417;
 
+  ImATeapot = 418;
```

Comment:
Should this be IAmATeapot, since we drop the apos? https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#418